### PR TITLE
Revision of Guide for Authors for #366

### PR DESCRIPTION
**Pull request by mpadge**
This starts the revision process outlined in #366.

### CODE REVIEW

File: softwarereview_author.Rmd [hunk 1]
```diff
@@ -4,23 +4,35 @@
 This concise guide presents the software peer review process for you as a package author.
 ```
 
+## Prior to Submission
```
Comment (maelle) [inline softwarereview_author.Rmd:7]: Maybe submission planning should be an item even before this one?

Comment (maelle) [reply softwarereview_author.Rmd:7]: ah that's the actual content. Maybe the title just needs to contain "planning"

File: softwarereview_author.Rmd [hunk 2]
```diff
@@ -4,23 +4,35 @@
...
+
+## Preparing for Submission
+
+-   Read and follow [our packaging style guide](#building), [reviewer's guide](#preparereview) to ensure your package meets our style and quality criteria.
+-   All submissions are automatically checked by our [pkgcheck](https://docs.ropensci.org/pkgcheck/) system to ensure packages follow our guidelines. All authors are expected to have run [the main `pkgcheck()` function](https://docs.ropensci.org/pkgcheck/reference/pkgcheck.html) locally to confirm that the package may be submitted.
+-   If there are any aspects of `pkgcheck` which your package is unable to pass, please explain reasons in your submission template.
+-   If you feel your package is in scope for the
+    [Journal of Open-Source Software](https://joss.theoj.org/) (JOSS), do not submit it to JOSS consideration until after the rOpenSci review process is over: if your package is deemed in scope by JOSS editors, only the accompanying short paper would be reviewed, (not the software that will have been extended reviewed by rOpenSci by that time). Not all rOpenSci packages will meet the criteria for JOSS.
+
```
Comment (maelle) [inline softwarereview_author.Rmd:27]: maybe suggest to ask any question in the rOpenSci forum

File: softwarereview_author.Rmd (hunk 1 above)
Comment (mpadge) [reply softwarereview_author.Rmd]: Rephrased to "Planning a submission" - thanks

File: softwarereview_author.Rmd (hunk 2 above)
Comment (mpadge) [reply softwarereview_author.Rmd:28]: done - thanks

File: softwarereview_author.Rmd [hunk 3]
```diff
@@ -4,23 +4,40 @@
 This concise guide presents the software peer review process for you as a package author.
 ```
 
--   Consult our [policies](#policies) see if your package meets our criteria for fitting into our suite and is not overlapping with other packages.
+## Planning a Submission
```
Comment (maelle) [inline softwarereview_author.Rmd:7]:
```suggestion
## Planning a Submission (or Pre-Submission Enquiry)
```

**Review by maelle — APPROVED** [review]
This looks great, thank you! Would you mind updating the changelog? Thank you!

File: softwarereview_author.Rmd (hunk 3 above)
Comment (mpadge) [reply softwarereview_author.Rmd]: Done, thanks

**Pull request merged**